packages/loki/package.json

```diff
@@ -18,6 +20,10 @@
     "android",
     "web"
   ],
+  "storybook": {
+    "displayName": "Loki",
+    "icon": "https://loki.js.org/img/favicon.svg"
+  },
```

Comment:
What about setting `unsupportedFrameworks` and `supportedFrameworks` accordingly?

Reply:
I had it from the beginning, but was thinking that it should already support all frameworks out of the box. Maybe it makes sense to add it anyways for things we've tried and tested with.